src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+         action="http://localhost:3000/api"
+         method="post"
+         name="test-form"
+         >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label
+             class="form-field">
+               Surname:
+          <input
+               type="text"
+               name="Surname"
+               required
+               autocomplete="off"
+          >
+        </label>
```

Comment:
please fix all places according to checklist
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="Surname"
            required
            autocomplete="off"
          >
        </label>
```